test/e2e/specs/onboarding/signup__free.ts

```diff
@@ -81,7 +81,20 @@ describe( DataHelper.createSuiteTitle( 'Signup: WordPress.com Free' ), function
 			if ( isGoalsStep ) {
 				await page.click( '.select-card__container:first-of-type' );
 				await startSiteFlow.clickButton( 'Continue' );
-			} else {
+			}
+		} );
+
+		it( 'Select a vertical', async function () {
+			const isVerticalsStep = await page.isVisible( '.site-vertical' );
```

Comment:
I wrote a long message in https://github.com/Automattic/wp-calypso/pull/64829/files#r904406971 before I realized the changes had been merged, but generally speaking, the `isVisible` approach is _very_ flaky and should not be relied upon.

Reply:
Note, if my request is unclear, let's drop the change request for now - in the interest of helping iterate quicker. 

I can work on moving the checks into the page objects, since that's one of my responsibilities.